### PR TITLE
Playtest Changes

### PR DESCRIPTION
**Pull request by sunjay**
All the changes we made during the playtest to be reviewed (at least in part) by both @sunjay and @kintel.

Looking at [style](https://github.com/anyWareSculpture/coding-style/blob/master/CODING_STYLE.md), maintainability, functionality, and adherence to flux principles (store vs. view, no setters outside actions, etc.).

### CODE REVIEW

File: src/actions/mole-game-action-creator.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+export default class MoleGameActionCreator extends BaseActionCreator {
+  // Action types
+  static ACTIVATE_PANEL = "activate-panel";
+  static DEACTIVATE_PANEL = "deactivate-panel";
+
+  /**
+   * Signals the mole game to activate a panel, typically from a timer
+   */
+  sendActivatePanel({stripId, panelId}) {
```
Comment (sunjay) [inline src/actions/mole-game-action-creator.js]: @kintel What is the difference between this and sendPanelPressed in panels-action-creator.js?

Comment (kintel) [inline src/actions/mole-game-action-creator.js]: It doesn't press the panel, it activates it (i.e. makes is touchable in the mole game).

We should probably do a sprint on terminology where we define "active" as well as panel vs. tile vs. strip.

File: src/config/default-config.js [hunk 2]
```diff
@@ -48,89 +49,119 @@ export default class DefaultConfig {
...
+    // any custom logic in any of the games
+    this.PANEL_DEFAULTS = {
+      ACTIVE_INTENSITY: 100,
+      INACTIVE_INTENSITY: 0
+    };
+
     /******* GAMES CONFIGURATION *******/
 
+    this.HANDSHAKE_GAME = {
+      TRANSITION_OUT_TIME: 4000
```
Comment (sunjay) [inline src/config/default-config.js]: @kintel Please add a comment here describing what exactly the purpose and usage of this setting is

File: src/actions/mole-game-action-creator.js (hunk 1 above)
Comment (sunjay) [inline src/actions/mole-game-action-creator.js]: For now though we should change this to something else like "available" or something. I don't know when we'll get to fixing this and it's easier to change it now before the merge.

File: src/animation/panel-animation.js [hunk 3]
```diff
@@ -67,8 +67,8 @@ export default class PanelAnimation {
    * Usually it isn't necessary to override this
    */
   playNextFrame() {
-    this.currentFrame++;
-    
+    this.currentFrame = this.currentFrame + 1;
```
Comment (sunjay) [inline src/animation/panel-animation.js]: Was the `++` from before causing problems?

Comment (kintel) [inline src/animation/panel-animation.js]: It's against out coding style: https://github.com/anyWareSculpture/coding-style/blob/master/CODING_STYLE.md#operators

Comment (sunjay) [inline src/animation/panel-animation.js]: You're right! Thanks for catching that! I sometimes forget when I switch between different projects. :smile:

**Pull request merged**